services/wallet/token/uniswap_tokenstore.go

```diff
@@ -0,0 +1,59 @@
+package token
+
+import (
+	"io/ioutil"
+	"net/http"
+	"time"
+)
+
+type uniswapStore struct {
+	client        *http.Client
+	tokensFetched bool
+}
+
+const uniswapTokensURL = "https://gateway.ipfs.io/ipns/tokens.uniswap.org" // nolint:gosec
+const tokenListSchemaURL = "https://uniswap.org/tokenlist.schema.json"     // nolint:gosec
+
+func NewUniswapStore() *uniswapStore {
+	return &uniswapStore{client: &http.Client{Timeout: time.Minute}, tokensFetched: false}
+}
+
+func (ts *uniswapStore) doQuery(url string) (*http.Response, error) {
+	resp, err := ts.client.Get(url)
```

Comment:
@IvanBelyakoff you could just do `return ts.client.Get(url)` without the part below.